internal/postgresConfig/get/get.go

```diff
@@ -0,0 +1,57 @@
+package get
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"github.com/spf13/afero"
+	"github.com/supabase/cli/internal/utils"
+	"io"
+	"os"
+	"text/tabwriter"
+)
+
+func Run(ctx context.Context, projectRef string, fsys afero.Fs) error {
+	// 1. get current config
+	{
+		config, err := GetCurrentPostgresConfig(ctx, projectRef)
+		if err != nil {
+			return err
+		}
+		PrintOutPostgresConfigOverrides(config)
+		return nil
+	}
+}
+
+func PrintOutPostgresConfigOverrides(config map[string]interface{}) {
+	fmt.Println("- Custom Postgres Config -")
+	const padding = 4
+	w := tabwriter.NewWriter(os.Stdout, 0, 0, padding, ' ', tabwriter.Debug)
+	fmt.Fprintf(w, "Config\tValue\t\n")
+	for k, v := range config {
+		fmt.Fprintf(w, "%s\t%+v\t\n", k, v)
+	}
+	w.Flush()
+	fmt.Println("- End of Custom Postgres Config -")
```

Comment:
We've been rendering markdown tables in most other commands, ie. `migration list`, `sso list`, etc. Shall we do the same here for consistency?

Reply:
done